src/test/scala/mesosphere/Builders.scala

```diff
@@ -0,0 +1,106 @@
+package mesosphere
+
+import java.util.concurrent.atomic.AtomicInteger
+
+import mesosphere.marathon.Protos.Constraint
+import mesosphere.marathon.{NewGroupEnforceRoleBehavior, Seq}
+import mesosphere.marathon.core.check.Check
+import mesosphere.marathon.core.health.HealthCheck
+import mesosphere.marathon.core.pod.{Network, PodDefinition}
+import mesosphere.marathon.core.readiness.ReadinessCheck
+import mesosphere.marathon.raml.{App, Apps, Resources}
+import mesosphere.marathon.state.RootGroup.NewGroupStrategy
+import mesosphere.marathon.state.{AbsolutePathId, AppDefinition, BackoffStrategy, EnvVarValue, KillSelection, PortDefinition, Role, RootGroup, Secret, Timestamp, UnreachableStrategy, UpgradeStrategy, VersionInfo}
+
+import scala.concurrent.duration.FiniteDuration
+
+/**
+  * Home for constructors of common test fixtures.
+  *
+  * Convention: new{EntityType}.{variant}, where variant describes some recipe for creating different variations of standard fixtures; for example, a command app definition.
+  *
+  * For the standard variant, `apply` is used.
+  *
+  * All builders return valid objects with the least amount of information required (IE - where empty list or None is valid, this is used as a default)
+  */
+object Builders {
+
+  object newRootGroup {
```

Comment:
I'm pretty sure there are duplicates in some tests somewhere.

Reply:
Yeah, I marked the other builder as deprecated. See here for how using this builder improves things:

https://github.com/mesosphere/marathon/pull/7150/commits/f0c1cc79ac2afaf94785981aa72ef059774d3c7a